--- default.kdl	2026-02-18 19:27:16.537689617 +0000
+++ config.kdl	2026-06-19 05:40:18.354027178 +0100
@@ -429,7 +429,7 @@
 // (Requires restart)
 // Default: true (if the host terminal supports it)
 // 
-// support_kitty_keyboard_protocol false
+support_kitty_keyboard_protocol true
 // Whether to make sure a local web server is running when a new Zellij session starts.
 // This web server will allow creating new sessions and attaching to existing ones that have
 // opted in to being shared in the browser.
