--- default.kdl	2026-08-28 17:17:59.313903226 +0100
+++ config.kdl	2026-08-28 17:33:24.380358874 +0100
@@ -625,7 +625,7 @@
 // in any pane - including one running on a remote machine over SSH - can
 // read the clipboard without the user being asked.
 // Default: false
-// dangerously_enable_paste_buffer_read false
+dangerously_enable_paste_buffer_read true
  
 // How to handle a nested Zellij session detected inside a pane.
 // Options:
